src/dpi.rs

```diff
@@ -126,19 +129,19 @@ impl Pixel for f64 {
 
 /// Checks that the DPI factor is a normal positive `f64`.
 ///
-/// All functions that take a DPI factor assert that this will return `true`. If you're sourcing DPI factors from
-/// anywhere other than winit, it's recommended to validate them using this function before passing them to winit;
-/// otherwise, you risk panics.
+/// All functions that take a DPI factor assert that this will return `true`. If you're sourcing DPI
+/// factors from anywhere other than winit, it's recommended to validate them using this function
+/// before passing them to winit; otherwise, you risk panics.
 #[inline]
 pub fn validate_hidpi_factor(dpi_factor: f64) -> bool {
     dpi_factor.is_sign_positive() && dpi_factor.is_normal()
 }
 
 /// A position represented in logical pixels.
 ///
-/// The position is stored as floats, so please be careful. Casting floats to integers truncates the fractional part,
-/// which can cause noticable issues. To help with that, an `Into<(i32, i32)>` implementation is provided which
-/// does the rounding for you.
+/// The position is stored as floats, so please be careful. Casting floats to integers truncates the
+/// fractional part, which can cause noticable issues. To help with that, an `Into<(i32, i32)>`
```

Comment:
This should discuss the issues, and emphasize that some issues (i.e. user-visible loss of precision) are not solved by using the `Into` impl.

Reply:
Huh, those docs are completely wrong now - they're stored generically, not as floats. Will revise, thanks.

Reply:
I've added a bit to the module docs.